CONTRIBUTING.md

```diff
@@ -53,18 +53,38 @@ You can run the tests with
 go test
 ```
 
+Making PRs
+----------
+
+All PRs should target the `next` branch as their base. This means that we can land them and stage them for a release without making multiple changes to `master` (which would cause multiple releases due to `go get`'s behaviour).
+
+The exception to this rule is for an urgent bug fix when `next` is already ahead of `master`. See [hotfixes](#hotfixes) for what to do then.
+
 Releasing a New Version
 -----------------------
 
 If you are a project maintainer, you can build and release a new version of
 `bugsnag-go` as follows:
 
-1. Commit all your changes.
-1. Update the version number in `bugsnag.go`.
-1. Add an entry to `CHANGELOG.md` and update the README if necessary.
-1. Commit tag and push
+#### Planned releases
+
+**Prerequisite**: All code changes should already have been reviewed and PR'd into the `next` branch before making a release.
+
+1. Decide on a version number and date for this release
```

Comment:
Prefer making bullet points complete sentences to make screen readers understand that they're different sentences.

Reply:
Do you just mean adding a full stop?

Reply:
Yes.